packages/docs/src/locales/en/en.json

```diff
@@ -2381,7 +2381,7 @@
       },
```

Comment:
Simple select still closes when I press `up` on first item. It shouldn't.
![image](https://user-images.githubusercontent.com/5394573/124173773-2d112980-dab4-11eb-91fe-8bc59101a2ae.png)

Reply:
Sorry, wasn't able to reproduce it.

For me it stays opened and do not close.

I've checked our code as well:

- Key Up from these "Options list" in this case emits
`this.$emit('no-previous-option-to-hover')`


- And this is the only thing we do in parent
`@no-previous-option-to-hover="focusSearchBar"`

focusSearchBar just focuses search bar, nothing more.

That's why described behavior seems really strange, may be you could try it again I think problem was in something else.